BT-Tracking/Interface/Current Data/CurrentDataVM.swift

```diff
@@ -72,11 +82,53 @@ final class CurrentDataVM {
         }
 
         /*if let lastFetchedDate = AppSettings.currentDataLastFetchDate {
-            var components = DateComponents()
-            components.hour = 3
-            if Calendar.current.date(byAdding: components, to: lastFetchedDate)! > Date() { return }
-        }*/
+         var components = DateComponents()
+         components.hour = 3
+         if Calendar.current.date(byAdding: components, to: lastFetchedDate)! > Date() { return }
+         }*/
```

Comment:
Asi uz by to chtelo smazat. Pokud to nekdy bude zase potreba, je to vzdy dohledatelne v gitu.